VirtoCommerce.Storefront.Model/Tax/TaxLine.cs

```diff
@@ -30,5 +27,7 @@ public TaxLine(Currency currency)
         /// </summary>
         public Money Price { get; set; }
         public string TaxType { get; set; }
+        public string TypeName { get; set; }
+        public ICollection<TaxDetail> TaxDetails { get; set; }
```

Comment:
better to use property assignment here  ` = new List`

Reply:
And change `ICollection` to `IList`.
Always use IList - it is shorter and allows to access items by index.

Reply:
ok, I' fix it